sql/2019/08_Security/08_25-34Sum.sql

```diff
@@ -0,0 +1,27 @@
+# standard SQL
+# 08_24-34Sum.sql
+#
+# Calculate usage percentage for each of the 10 metrics 08.25-34 metrics in one run
+#
+# BigQuery usage notes: 
+# == archive.summary_response_bodies =  71.5 GB usage (firstHtml partitioning)
+# == archive.summary_requests = 117.5 GB 
+#  
+# cross-origin-opener-policy - only 6 instances of this data across the archive
+# sec-fetch-* - zero instances of this data across the archive
+
+SELECT

```

Comment:
One other thing that we're trying to do now is split by desktop/mobile (for context: https://httparchive.slack.com/archives/CGFLV2S06/p1565208006011800).

The partitioned tables include a `client` field that you can add here and to the `GROUP BY` (all queries)